src/components/DpSelect/DpSelect.stories.mdx

```diff
@@ -0,0 +1,121 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import DpSelect from './DpSelect'
+
+<Meta
+  title="Components/Select"
+  component={DpSelect} />
+
+# DpSelect
+
+The DpSelect component is used to render a select input field with customizable options. It provides functionality for selecting an option from a dropdown list and emits a select event with the selected value.
+
+## Default usage
+
+```html
+<dp-select
+  v-model="selectModel"
+  :label="{
+          text: Translator.trans('Text')
+        }"
+  name="select_menu_name"
+  :options="selectOptions"
+  required
+  @select="updateLocalState" />
+```
+
+export const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: { DpSelect },
+    template: `<dp-select v-bind="$props" />`
+});
+
+### Classes
+<Canvas>
+    <Story
+        name="Classes"
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
+            },
+            classes: 'layout__item u-6-of-12 sbdocs-preview css-tmvzag'
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Disabled
+
+<Canvas>
+    <Story
+        name="Disabled"
+        args={{
+            id: 'selectId',
+            label: {
+              text: 'select label'
```

Comment:
For all examples: i think this label is a better fit, especially for the options below.

```suggestion
              text: 'Chose a number'
```